electron-app/src/electron-main.ts

```diff
@@ -41,7 +41,7 @@ const createWindow = (): void => {
   });
 
   // Open the DevTools.
-  // mainWindow.webContents.openDevTools();
+  mainWindow.webContents.openDevTools();
```

Comment:
If you commit this one more time I will find a way to steal your keyboard and ban you from making changes to the project